evil-escape.patch

```diff
@@ -1,8 +0,0 @@
---- a/evil-escape.el
-+++ b/evil-ewcape.el
-@@ -1,4 +1,4 @@
-- ;;; evil-escape.el --- Escape from anything with a customizable key sequence
-+;;; evil-escape.el --- Escape from anything with a customizable key sequence
```

Comment:
This patch looks really useless, but just to make sure that this rename is not accidental I decided to try if `evil-escape` still worked after the removal of this patch, and it seems it works without problems.